addons/stock_dropshipping/models/sale.py

```diff
@@ -15,7 +15,7 @@ def _get_qty_procurement(self, previous_product_uom_qty):
         if purchase_lines_sudo.filtered(lambda r: r.state != 'cancel'):
             qty = 0.0
             for po_line in purchase_lines_sudo.filtered(lambda r: r.state != 'cancel'):
-                qty += po_line.product_uom._compute_quantity(po_line.product_qty, self.product_uom, rounding_method='HALF-UP')
+                qty += po_line.product_uom._compute_quantity(po_line.product_qty, po_line.product_uom, rounding_method='HALF-UP')
```

Comment:
Because you call `_compute_quantity` with the same UoM, the `_compute_quantity` is useless then (It rounds only), no ?

Reply:
Yes indeed, thanks !